.gitignore

```diff
@@ -38,3 +38,4 @@ transaction-logs
 secrets.yml
 .gradletasknamecache
 .sts4-cache
+out/
```

Comment:
I am not super keen to add such a global pattern. Perhaps we would make that a bit more specific? See also https://github.com/spring-io/initializr/pull/657/files

Reply:
That one's a quite common `.gitignore` entry for IntelliJ. For instance, see GitHub's own [Jetbrains `.gitignore` template](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore#L33). That being said, I'm fine with it being updated to something more specific if you prefer.